spec/rubocop/cop/layout/empty_line_after_guard_clause_spec.rb

```diff
@@ -28,7 +34,7 @@ def foo
       end
     RUBY
 
-    expect_correction(<<~RUBY)
+    expect_correction(<<~RUBY )
```

Comment:
That's just my two cents.

```suggestion
    expect_correction(<<~RUBY)
```

Reply:
🤦 thanks @koic!